clinica/pydra/dwi/_commons/_workflows.py

```diff
@@ -0,0 +1,153 @@
+from typing import List
+
+from pydra import Workflow
+
+
+def build_eddy_fsl_workflow(
+    compute_mask: bool = True,
+    image_id: bool = False,
+    field: bool = False,
+    output_dir=None,
+    name: str = "eddy_fsl",
+) -> Workflow:
```

Comment:
In accordance with the docstring:

```suggestion
from typing import List, Optional

from pydra import Workflow


def build_eddy_fsl_workflow(
    compute_mask: bool = True,
    image_id: bool = False,
    field: bool = False,
    output_dir: Optional[str] = None,
    name: str = "eddy_fsl",
) -> Workflow:
```